packages/solutions/module-tools/package.json

```diff
@@ -54,9 +54,9 @@
     "@babel/types": "^7.21.5",
     "@babel/traverse": "^7.21.5",
     "@modern-js/core": "workspace:*",
-    "@modern-js/libuild": "0.13.1",
-    "@modern-js/libuild-plugin-svgr": "0.13.1",
-    "@modern-js/libuild-plugin-swc": "0.13.1",
+    "@modern-js/libuild": "0.13.2",
+    "@modern-js/libuild-plugin-svgr": "0.13.2",
+    "@modern-js/libuild-plugin-swc": "0.13.2",
```

Comment:
The dependency of `@modern-js/plugin-module-import` should be updated too.

![image](https://github.com/web-infra-dev/modern.js/assets/7237365/75527f77-7651-44b8-afb7-93494a7c09b5)